plugins/MultiSites/angularjs/dashboard/dashboard-model.service.js

```diff
@@ -66,6 +66,57 @@
                 site.visits_evolution    = parseInt(site.visits_evolution, 10);
                 site.pageviews_evolution = parseInt(site.pageviews_evolution, 10);
                 site.revenue_evolution   = parseInt(site.revenue_evolution, 10);
```

Comment:
@bx80 This seems to cause a problem with the evolution. For some randomly generated data it shows:
![image](https://user-images.githubusercontent.com/1579355/139816206-a0aec75c-adaf-4cb0-a7b8-125ef6c91797.png)
But comparing 379 to 15 visits would more be like a 2343% evolution, wouldn't it?
I would assume this might be some kind of formatting glitch. We often had that when values are formatted twice. 
It seems the API returns it already formatter and the UI tries to format it again here, causing everything after the thousand separator to be cut of. Actually I'm not sure why we try to format it again here. Maybe we can simply remove any formatting, if API already does that.

Reply:
I wasn't able to recreate this with my data, I had 1748 visits so far this month vs 355 last month, ((1748-355)/355) = 392% which is what was displayed. Clearly there is some issue here as your screenshot should be showing something like 2426% ((379-15)/15). I tried removing the parseInt() formatting, but it looks like the fancy coloring and up/down arrows on the evolution column need it in this format, so instead I moved this formatting code after the tooltip is generated. 

Actually just wondering if the issue is that your test data percent is more than 999 so a euro style thousand separator is added by the API formatting giving 2.426,00 then parseInt reads the . as a decimal place giving 2 point 426, which rounds to 2% ? Wild guess, but if that is the case then my last commit should fix the tooltip but the evolution value on the row would still be wrong, as that would be an existing bug not caused by this change.

Reply:
Recreated! Looks like any percent higher than 999% will be broken by parseInt()

![image](https://user-images.githubusercontent.com/88810029/140254356-e4498357-bbd1-41c3-b988-be6f92fe4525.png)

It looks like parseInt() is being used so that the site.directive.html can do a plus/zero/minus comparison to decide which arrows to show, so it can't just be removed. Instead I've committed a fix where the parseInt() values are stored in different model properties instead of overwriting the API formatted values, these new _int values are then used to decide on the arrow to use. Since we only need to know if the value is positive, zero or negative then the decimal places and parts after the thousand separator shouldn't matter. The original API formatted values can then be shown without being mangled by parseInt().